db/seeds/gear.seeds.rb

```diff
@@ -1,103 +1,103 @@
+    { name: 'Plain Pickup', tags: '', playbooks: [@wronged], keywords: 'transport' },
+    { name: 'Plain Van', tags: '', playbooks: [@wronged], keywords: 'transport' },
+    { name: 'Pocket Knife', harm: 1, tags: 'hand, useful, small', playbooks: [@mundane] },
+    { name: 'Protective Wear', armor: 1, tags: '', playbooks: [@wronged] },
+    { name: 'Razor Whip', harm: 3, tags: 'hand, area, messy, holy', playbooks: [@divine] },
+    { name: 'Ritual Knife', harm: 1, tags: 'hand', playbooks: [@spell_slinger] },
+    { name: 'Sawn-off Shotgun', harm: 3, tags: 'hand/close, messy, loud, reload', keywords: 'signature', playbooks: [@wronged] },
+    { name: 'Shotgun', harm: 3, tags: 'close, messy, loud', keywords: 'signature', playbooks: [@expert, @flake, @wronged] },
+    { name: 'Shotgun', harm: 3, tags: 'close, messy', keywords: 'normal, modern', playbooks: [@crooked, @initiate, @monstrous, @professional, @spooky] },
+    { name: 'Silver Knife', harm: 1, tags: 'hand, silver', playbooks: [@initiate], keywords: 'old-fashioned' },
+    { name: 'Silver Sword', harm: 2, tags: 'hand, messy, silver', playbooks: [@expert] },
+    { name: 'Silver Trident', harm: 3, tags: 'hand, silver, holy', playbooks: [@divine] },
+    { name: 'Skateboard', tags: '', playbooks: [@mundane], keywords: 'transport' },
+    { name: 'Sledgehammer', harm: 3, tags: 'hand, messy', playbooks: [@mundane] },
+    { name: 'Small Handgun', harm: 2, tags: 'close, loud, reload', playbooks: [@mundane] },
+    { name: 'Sniper Rifle', harm: 4, tags: 'far', keywords: 'serious', playbooks: [@professional] },
+    { name: 'Sniper Rifle', harm: 3, tags: 'far', keywords: 'serious, modern', playbooks: [@initiate] },
+    { name: 'Spear', harm: 2, tags: 'hand/close', playbooks: [@initiate], keywords: 'old-fashioned' },
+    { name: 'Specialist Weapon (Other Foe)', harm: 1, tags: "Keeper's Discretion", keywords: 'signature', playbooks: [@wronged] },
+    { name: 'Specialist Weapons (Chosen Foe)', harm: 4, tags: "Keeper's Discretion", keywords: 'signature', playbooks: [@wronged] },
+    { name: 'Submachine Gun', harm: 2, tags: 'close, reload, area', playbooks: [@crooked] },
+    { name: 'Submachine Gun', harm: 3, tags: 'close, area, loud, reload', keywords: 'serious', playbooks: [@professional] },
+    { name: 'Sword', harm: 2, tags: 'hand, messy', playbooks: [@initiate, @monstrous], keywords: 'old-fashioned' },
+    { name: 'Throwing Knives', harm: 1, tags: 'close, many', playbooks: [@flake] },
+    { name: 'Thunder Hammer', harm: 3, tags: 'hand, stun, holy', playbooks: [@divine] },
+    { name: 'Truck', tags: '', keywords: 'transport', playbooks: [@professional] },
+    { name: 'Van', tags: '', keywords: 'transport', playbooks: [@mundane, @professional] },
+    { name: "Watchman's Flashlight", harm: 1, tags: 'hand', playbooks: [@flake] },
+    { name: 'Weighted Gloves/Brass Knuckles', harm: 1, tags: 'hand', playbooks: [@flake] }
```

Comment:
So, the issue here is that the playbooks don't know what categories they want, so it's always going to pick whatever category comes first (or last, if you're overwriting).  So if you're populating, say, a 9mm, that has multiple categories it can fit into (Normal, Modern, and Practical, in this case), all PlaybookGear entries for 9mm will categorized as normal, even if the referenced playbook does not have that category.  My fix for this would be to include an acts_as_taggable on Playbook tagging the categories that the playbook itself includes and then using that in the seed file to distinguish which categories it's looking for.

Reply:
Added below.